packages/menu/src/SubMenu/SubMenu.tsx

```diff
@@ -70,21 +72,36 @@ export const SubMenu = InferredPolymorphic<SubMenuProps, 'button'>(
       children,
       onClick,
       description,
-      setOpen,
       className,
       glyph,
       onExited = () => {},
-      open = false,
+      open: openProp = false,
+      setOpen: setOpenProp,
       active = false,
       disabled = false,
       size = Size.Default,
       as,
       ...rest
     },
-    ref: React.Ref<any>,
+    fwdRef: React.Ref<any>,
   ): React.ReactElement => {
     const { Component } = useInferredPolymorphic(as, rest, 'button');
-    const { theme, darkMode } = useContext(MenuContext);
+    const {
+      theme,
+      darkMode,
+      highlightIndex: _highlightIndex,
+    } = useContext(MenuContext);
+    const { index: _index, ref } = useDescendant(
```

Comment:
are these `_*` variables getting used later? if not, maybe don't need to destructure them

Reply:
Will be used later